chapter1/item2.md

```diff
@@ -2,4 +2,56 @@
 
 *This item covers customization using X resource files and tools*
 
-TODO
+---
+
+X resources provide a simple yet powerful way to customize your X session. With files such as `.Xresources`, you can easily assign colors and fonts globally or for specific applications. 
+
+_NB_: If you have a `.Xdefaults` file as opposed to a `.Xresources`, it may be a good idea to change it. `.Xdefaults` is reloaded everytime an X program starts, therefore less eddicient.
+
+Most often, it is used to set colors globally and then application specific options, typically for a terminal. `urxvt` is a common item that has its options set in `.Xresources`. The syntax is quite simple,
```

Comment:
Um, no. Xresources aren't used tipically for terminals. A lot of applications (even GUI ones) will make use of it. Xresources is related to uniform configuration across applications, not only colors in terminal.